data/db/mil-om-system.xml

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE lensdatabase SYSTEM "lensfun-database.dtd">
+<lensdatabase version="1">
+
+    <mount>
+        <name>Micro 4/3 System</name>
+        <compat>M42</compat>
+        <compat>M39/1</compat>
+        <compat>DKL</compat>
+        <compat>T2</compat>
+        <compat>Generic</compat>
+    </mount>
+    
+
+    <lens>
+        <maker>OM</maker>
+        <model>OM 20mm F1.4</model>
+        <mount>Micro 4/3 System</mount>
+        <cropfactor>2.00308</cropfactor>
```

Comment:
@sarunasb good question ... I believe I copied it from somewhere but since it's 2 months ago, I don't remember anymore. 🤷🏻  I have no problem changing it to 2. :-)

Reply:
You seem to want to add a new XML file for OM System. In such case `<mount>` element for Micro 4/3 System can not be included in the new file. Mount values serve as a unique index. µ4/3 mount is already in mil-olympus.xml. So either continue adding OM cameras and lenses to Olympus, or have om-system.xml without a mount definition...

Reply:
@sarunasb oh i see. another option coming to my mind would be to rename the existing file `mil-olympus.xml` to `mil-om-system.xml`? at the end of the day, i am perfectly fine following your guidance as this is my first contribution to this project. :-)

Reply:
Renaming the existing file would be confusing. 

> Why does darktable still use version 1 of lensfun. Lens in version 1 don't get updated.

That's incorrect. The version 2 database is converted to version 1 automatically, and when you run `lensfun-update-data` it downloads both versions with identical contents.

Reply:
> @sarunasb oh i see. another option coming to my mind would be to rename the existing file `mil-olympus.xml` to `mil-om-system.xml`?

No, that's even more confusing.

Reply:
@Sturmflut ok, so what would be your preference? :) Add to `mil-olympus.xml`?

Reply:
@Sturmflut ok with this suggestion? https://github.com/lensfun/lensfun/pull/1523#issuecomment-1046285240

Reply:
I agree with @sarunasb (see comment below). Remove the ` <mount>` block and rename to `om-system.xml`.

Reply:
@Sturmflut ok, done.